system/Session/Session.php

```diff
@@ -511,7 +511,22 @@ public function get(string $key = null)
 
 		return $userdata;
 	}
-
+	/**
+	 * Push a value into a session array.
+	 *
+	 * @param  string  $key
+	 * @param  mixed   $value
+	 * @return void
+	 */
+	public function push($key, $value)
+	{
+		if ($this->has($key)) 
```

Comment:
Please change to an early out design: 

```
if (! $this->has($key)) return;

$array = $this->get($key);
$array = array_merge($array, $value);
$this->set($key, $array);
```

Reply:
ok

Reply:
not getting my way with the test, not so go at it. mind putting me true?? i use windows.

Reply:
HAve you read https://github.com/bcit-ci/CodeIgniter4/tree/develop/tests

Reply:
@jaynarayan89  tried that, i keep getting no test executed.